docker/package/tezos_setup_wizard.py

```diff
@@ -225,12 +225,10 @@ def wait_for_ledger_baking_app():
             f"sudo -u tezos {suppress_warning_text} tezos-client list connected ledgers"
         ).stdout
         proc_call("sleep 1")
-    base_ledger_url = (
-        re.search(b"ledger:\/\/[\w\-]+\/", output).group(0).decode("utf-8")
+    ledgers = list(
+        {url.decode() for url in re.findall(b"ledger:\/\/[\w\-]+\/", output)}
     )
-    return base_ledger_url, [
-        name.decode() for name in re.findall(derivation_path_regex, output)
-    ]
+    return ledgers, [name.decode() for name in re.findall(ledger_regex, output)]
```

Comment:
Ahh, I think, if I got what you wanted to do in `ledger_urls_info` right (see my previous comment), you missed gluing every derivation together with its ledger address here, otherwise `ledgers_derivations` contains the same incomplete paths as `ledger_derivation` before, i.e. the ones you had to combine with `base_ledger_url` we no longer have.
🤔